witchcraft-server/src/lib.rs

```diff
@@ -251,6 +251,23 @@
 //! * `process.filedescriptor` (gauge) - The number of file descriptors held open by the process divided by the maximum
 //!     number of files the server may hold open.
 //!
+//! ## Tokio
+//!
+//! * `tokio.tasks` (gauge) - The number of tasks in the Tokio runtime.
+//!
+//! ### Unstable metrics
+//!
+//! Services can opt-in to metrics that depend on unstable Tokio features. To do so, *both* enable the `tokio_unstable`
+//! Cargo feature of the `witchcraft-server` crate *and* set the [`tokio_unstable` cfg].
+//!
+//! * `tokio.blocking.threads` (gauge) - The number of threads in Tokio's blocking pool.
+//! * `tokio.blocking.threads.idle` (gauge) - The number of threads in Tokio's blocking pool that are idle.
+//! * `tokio.tasks.polls` (gauge) - The number of individual poll calls to tasks.
```

Comment:
This is mostly useful to normalize the values of `tokio.tasks.poll-duration-bucket` into percentages.